src/DynamoCore/Services/StabilityTracking.cs

```diff
@@ -100,6 +118,7 @@ public static void WriteUptimeBeat(TimeSpan timespan)
         /// </summary>
         public static void Startup()
         {
+            StabilityUtils.IsLastShutdownClean = IsLastShutdownClean();
```

Comment:
Should we do this (just wondering the purpose of yet another `static` class)? I'm imagining the same kind of things can be done by making `static bool IsLastShutdownClean` an `internal` method.